addons/digest/models/digest.py

```diff
@@ -65,9 +65,10 @@ def _compute_kpi_res_users_connected_value(self):
             record.kpi_res_users_connected_value = user_connected
 
     def _compute_kpi_mail_message_total_value(self):
+        discussion_subtype = self.env.ref('mail.mt_comment')
```

Comment:
As this is only used once to get its id, you could directly get the id.
`discussion_subtype_id = self.env.ref('mail.mt_comment').id`

Reply:
Done. :+1: